editor/src/components/editor/actions/actions.tsx

```diff
@@ -5162,6 +5163,8 @@ export const UPDATE_FNS = {
       imageDragSessionState: action.imageDragSessionState,
     }
   },
+  APPLY_COMMANDS: (action: ApplyCommandsAction, editor: EditorModel): EditorModel =>
+    foldAndApplyCommandsSimple(editor, action.commands),
```

Comment:
super minor: please use the { return ... } syntax, so it's easier to inject a console.log for debugging etc